### PR TITLE
Fix installation for Linux

### PR DESCRIPTION
**Pull request by Neelix57**
**- ./build.sh:** Commented some lines in order for me to better read the code.

**- src/tools/build.sh:** Maybe the /x I removed from the destination folder is MacOS specific ? Let me know.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -131,21 +131,21 @@ function create_unvars_c {
...
+# *                                                                                      *
+# *                        Reset Global Variables .c file                                *
+# *                                                                                      *
+# *            This is a stub file - actual variables are in vars.h.                     *
+# *        (this is autogenerated by the build script. do not hand edit.)                *
+# \**************************************************************************************/
 
 #define IN_UNVARS_C 1
-// include all the includes we'll (might) need (and want)
+# include all the includes we'll (might) need (and want)
```
Comment (rayarachelian) [inline build.sh:148]: Actually this is creating a .C file, though it's inside a bash shell script, therefore the comment should infact be // not #

File: src/tools/build.sh [hunk 2]
```diff
@@ -301,7 +301,7 @@ strip_and_compress $(for i in $SRCLIST; do echo ${i}${EXT}; done)
 ###########################################################################
 
 if [[ -n "$INSTALL" ]]; then
-      cd "${XTLD}/bin/x/$MACOSX_MAJOR_VER"
```
Comment (rayarachelian) [inline src/tools/build.sh:304]: Thanks for finding this typo

File: build.sh (hunk 1 above)
Comment (Neelix57) [reply build.sh:148]: Ok; I didn't know that But the # instead didn't cause any problem during compilation. Is it normal ?

File: src/tools/build.sh (hunk 2 above)
Comment (Neelix57) [reply src/tools/build.sh:304]: No problem I'm happy to help on such free projects

**Comment by Neelix57** [comment]
Added desktop launcher for Linux

**Comment by rayarachelian** [comment]
> Added desktop launcher for Linux

Much appreciated, will add this and the rest of your changes to the next unstable, and push, hopefully this weekend.
I'll try to test in Windows and various Mac as well.

**Comment by rayarachelian** [comment]
fixed windows compilation, retesting under linux, macos, etc. once test cycle is complete will push to unstable

**Comment by rayarachelian** [comment]
just pushed to unstable, these and a few other issues should be fixed now.
I merged your changes locally, so not merging via github. Also gave you credit in the ChangeLog file.

**Pull request closed without merging**